### DISCUSSION TITLE
Starting at specified directory in Gentoo

### DISCUSSION THREAD

**Discussion by Drogobo** [kovidgoyal/kitty#6168] [Q&A]
I use the exact same configs in Gentoo as I use in Arch. For some reason, Arch will always start at ~ or if I launched from a file manager, it will start at the right directory. This almost works on Gentoo, but for some reason, if I use my shortcut (super+enter) or start from terminal, it will start at /. I read the friendly manual, and it says it is a ENV error. I have no idea why it does this in Gentoo and not Arch; it seems to do different things. I am using the exact same configs on Gentoo and Arch, so it is odd. What do I have to do to make it work?

**Comment by kovidgoyal** [comment]
Unless you specify the --directory command line flag kitty will start
in whatever directory its parent process is in when launched. This has
nothing to do with the kitty config and everything to do with where and
how you are running kitty. If you are running it via a .desktop file
then the desktop file can change the cwd. If you are running it via some
kind of desktop environment launcher, then the cwd is typically whatever
the launcher thinks it should be, and so on.

**Reply by Drogobo** [reply]
Well, how do I set the directory in KDE? Again, oddly, it works on Arch but not Gentoo with the exact same configs. Specifically, I need the "Open terminal here" option in Dolphin to continue working, so just cding into ~ at startup would break that.

**Comment by Drogobo** [comment]
Gentoo (broken functionality): https://www.youtube.com/watch?v=o66rFQjNyEk (2nd time it starts at / which is bad)
Arch (expected functionality): https://www.youtube.com/watch?v=N_xCIU-QOSU (they all start where they are supposed to)
